app/upgrades/v19_2/upgrades_test.go

```diff
@@ -144,10 +144,6 @@ func TestAddCodeToERC20Extensions(t *testing.T) {
 
 			ctx := network.GetContext()
 
-			// check code does not exist
-			code := network.App.EvmKeeper.GetCode(ctx, expCodeHash)
-			require.Len(t, code, 0)
-
```

Comment:
Note that this was removed because the code corresponding to the ERC-20 code hash is now registered already on genesis because it's being added for the native `aevmos` token pair for the STR v2.